Code/GraphMol/RGroupDecomposition/RGroupDecompParams.h

```diff
@@ -74,6 +74,8 @@ struct RDKIT_RGROUPDECOMPOSITION_EXPORT RGroupDecompositionParameters {
   bool allowNonTerminalRGroups = false;
   // unlabelled core atoms can have multiple rgroups
   bool allowMultipleRGroupsOnUnlabelled = false;
+  // allow tautomeric core(s)
+  bool allowTautomerCore = false;
```

Comment:
Consider making this the default (i.e. allowTautomerCore = true)

Reply:
Works for me, but given that it will change the behavior RGD it would be best if you and/or Greg make the call,

Reply:
I'm very wary of changing default behavior unless the defaults produce confusing results.
Given that most RDKit code does not behave in a tautomer-insensitive way, I'd rather not make it the default here.

If it turns out to create problems, we can change the default in a later release.